composer.json

```diff
@@ -90,9 +90,6 @@
         "symfony/yaml": "^4.4 || ^5.4 || ^6.0",
         "vimeo/psalm": "^4.9.2"
     },
-    "conflict": {
-        "doctrine/doctrine-bundle": ">=3"
```

Comment:
I just saw this conflict and don't see why it's here.

Doctrine/Bundle is not used by this bundle (it's not even in the dependency-dev).

Reply:
I saw it too yesterday, when was added?

Reply:
https://github.com/sonata-project/SonataAdminBundle/pull/5759

But we don't rely on it anymore https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Resources/config/commands.php